example/sample1/federatedjob-template.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: core.federation.k8s.io/v1alpha1
+kind: FederatedJob
+metadata:
+  name: test-job
+  namespace: test-namespace
+spec:
+  template:
+    spec:
+      parallelism: 1
+      selector:
+        matchLabels:
+          app: nginx
+      manualSelector: true
+      template:
+        metadata:
+          labels:
+            app: nginx
+        spec:
+          terminationGracePeriodSeconds: 0
+          restartPolicy: Never
+          containers:
+            - name: nginx
+              image: nginx
```

Comment:
Please use `busybox` if all we're doing is looping for a while to keep the container running.

Reply:
done